docs/update_and_migration/from_3.3/update_from_3.3.md

```diff
@@ -418,4 +420,8 @@ Run the following scripts:
 
     ``` sql
     psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-3.3.33-to-3.3.34.sql
-    ```
\ No newline at end of file
+    ```
+
+### v3.3.40
+
+A command dealing with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be provided here in the near future.
```

Comment:
A command to deal with the duplicated entries caused by [IBX-8562](https://issues.ibexa.co/browse/IBX-8562) will be available soon.